README.md

```diff
@@ -34,34 +34,66 @@ Composer
 
 ### Database technology & provider
 
-MySQL
+CouchDB
 
 ## Repos and projects
 
 Client of the API: https://github.com/code4romania/rvm-client
 
-## Development
-
-    # To bootstrap the project (Run this only once), run the following commands in your shell:
 
-    ./dev/composer.sh update
-    cp src/.env.example src/.env
-    docker-compose up
-    ./dev/key_generate.sh
-    ./dev/artisan.sh migrate
-    ./dev/artisan.sh passport:install
-
-    # Every other time
+## Development
 
-    docker-compose up
+### To bootstrap the project (Run this only once), run the following commands in your shell:
+
+Install composer dependencies
+```
+docker run -v ${PWD}/src:/app -w /app composer:latest composer install --ignore-platform-reqs --no-scripts --no-interaction --prefer-dist --optimize-autoloader
+```
+
+Copy environment variables and start the application
+```
+cp src/.env.example src/.env
+docker-compose up
+```
+
+Setup CouchDB single-node here: http://localhost:5984/_utils#setup or do it manually by running:
+```
+curl -X PUT http://admin:admin@localhost:5984/_users
+curl -X PUT http://admin:admin@localhost:5984/_replicator
+curl -X PUT http://admin:admin@localhost:5984/_global_changes
+```
+
+Run database migrations
+```
+php ./src/database/migrations/v_1_0_0.php -c migrate
+```
+
+Generate app secret key, and personal access client keys
+```
+./scripts/artisan.sh key:generate
+./scripts/artisan.sh optimize
+./scripts/artisan.sh passport:install
+```
+
+Seed the database
+```
+docker run -v ${PWD}/src:/app -w /app composer:latest composer dump-autoload
```

Comment:
I don't think we need this line anymore

Reply:
We do sometimes. Say you come from master branch as it is right now, the Seeder won't be loaded. It doesn't hurt having it but could hurt if we don't 🤕